Lib9c/Action/Buy.cs

```diff
@@ -385,6 +385,11 @@ public override IAccountStateDelta Execute(IActionContext context)
 
                 if (!sellerAvatarState.inventory.RemoveTradableItem(tradableItem, count) && !fromLegacy)
                 {
+                    var msg = $"Invalid Buy Tx Found. TxId: {context.TxId}, Index: {context.BlockIndex}, ItemType: {purchaseInfo.itemSubType}, " +
+                              $"ProductId: {productId}, " +
+                              $"BuyerAgentAddress: {context.Signer}, BuyerAvatarAddress: {buyerAvatarAddress}, SellerAgentAddress: {sellerAgentAddress}, SellerAvatarAddress: {sellerAvatarAddress}" +
+                              $"Item Price: {shopItem.Price}, Tax: {tax}, Taxed Price: {taxedPrice}.";
```

Comment:
Is there any possibility about `shopItem` is `null`?

Reply:
No, if shop item is null,it will be fail on the upper line.

Reply:
🆗